corehq/apps/app_manager/views/view_generic.py

```diff
@@ -71,6 +71,12 @@ def view_generic(request, domain, app_id=None, module_id=None, form_id=None,
     except ModuleNotFoundException:
         return bail(request, domain, app_id)
 
+    if app and app.application_version == '1.0':
+        return render(request, 'app_manager/no_longer_supported.html', {
```

Comment:
Might be a good place for a soft_assert, so we get notified even if they don't contact support.